source/qdk_package/pyproject.toml

```diff
@@ -14,11 +14,11 @@ dependencies = ["qsharp==0.0.0"]
 
 [project.optional-dependencies]
 jupyter = ["qsharp-widgets==0.0.0", "qsharp-jupyterlab==0.0.0"]
-azure = ["azure-quantum>=3.2.0"]
+azure = ["azure-quantum>=3.5.0.dev0"]
```

Comment:
I would like to wait until an official release of the azure-quantum package before merging this PR in.